.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
Personally, I like `trim_trailing_whitespace`, but I don't have strong feels on it